v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/modbus_evse_client.py

```diff
@@ -337,7 +366,10 @@ async def start_charge_with_power(self, kwargs: dict, *args, **fnc_kwargs):
             )
             return
 
-        charge_power_in_watt = int(float(kwargs["charge_power"]))
+        charge_power_in_watt = parse_to_int(kwargs.get("charge_power", None), None)
+        if charge_power_in_watt is None:
+            self.__log(f"invalid charge_power: {kwargs.get('charge_power', None)}")
```

Comment:
Suggestion: put the result of `kwargs.get('charge_power', None)` in a local variable, so it does not have to be repeated.

Reply:
👍